aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -7,6 +7,27 @@
 import base64
 from botocore.errorfactory import ClientError
 
+HELICS_RUN = '''sudo /home/ubuntu/install-and-run-helics-scripts.sh
```

Comment:
Make sure to update against what is already on AWS when committing. There has been a habit of letting these slip out of sync

Reply:
at least currently it is in-sync